### ISSUE TITLE
Decorative images do not have ALT="" in cards tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1914]
URL: http://clarity-ks.surge.sh/cards

Reproduced Steps:
1. Go to Cards page
2. Use WAT2.0 to show “alt” of the images.

Actual Results:
The images have no alt in cards tab.

Expected Results:
The decorative images should have meaningful “alt”.

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/e120092e-e43b-4ffc-986c-3fb9781698fc)

NOTE: This may also be an issue on the website. Please check.
